app/views/users/show.html.erb

```diff
@@ -24,16 +24,32 @@
     <section id="github-followers">
       <h1>Github Followers</h1>
       <% current_user.list_followers.each do |follower| %>
-        <%= link_to follower.name, follower.link %> <br>
+        <span id=<%= follower.name%> >
+          <%= link_to follower.name, follower.link %> 
+          <% if current_user.potential_friends.keys.include?(follower.name) %>
+            <%= button_to "Add as Friend", "/friendships/#{current_user.id}/#{current_user.potential_friends[follower.name]}", method: :post %> 
+          <% end %> <br> 
+        </span>
       <% end %>
     </section>
 
     <section id="github-following">
       <h1>Github Following</h1>
       <% current_user.list_following.each do |following| %>
-        <%= link_to following.name, following.link %> <br>
+        <%= link_to following.name, following.link %> 
+        <% if current_user.potential_friends.include?(following.name) %>
+          <%= button_to "Add as Friend", "/friendships/#{current_user.id}/#{current_user.potential_friends[following.name]}", method: :post %> 
```

Comment:
clicking the button `Add as friend` creates a new `Friendship` in our `Friendships` table